### PR TITLE
Check for multiple blank lines.

### LINKED ISSUE #182: Check for multiple blank lines?
Is there a reason there is no sniff for multiple blank lines? It happens to me all the time that I accidentally commit stuff like this:

```php
public function myTest()
{
    debug('a thing);


    debug('another thing')
}
```

### PR DESCRIPTION
**Pull request by ADmad**
Closes #182

### CODE REVIEW

**Comment by dereuromark** [comment]
Cool!

**Comment by ADmad** [comment]
Hmm actually `<property name="ignoreBlankLines" value="false"/>` mean it should already check for blank lines. So not sure if #182 is invalid or there's some issue. Closing for now.

**Comment by bravo-kernel** [comment]
Thanks for picking this up 👍

**Comment by gmponos** [comment]
I would like also to add that in `Codesniffer` I haven't seen any sniff that disallows the multiple lines inside classes.

So this might be still allowed:

```
class MyClass
{
    public $test;


    public $test2;
}
```

You can add a custom sniff [like this](https://github.com/webthinkgr/codesniffer/blob/master/Webthink/Sniffs/WhiteSpace/ClassEmptyLinesSniff.php) to prevent it I can submit a related MR if you wish.

**Comment by cleptric** [comment]
@gmponos I debugged that inside a class yesterday to find the cause. I also added a test for that now :)

**Comment by gmponos** [comment]
> I think it is also allowed between functions like this:

```
class MyClass
{
    public function myFunction()
    {
    // No body
    }


    public function myFunction2()
    {
    // No body
    }
}
```

**Comment by cleptric** [comment]
I added a new [sniff](https://github.com/spryker/code-sniffer/blob/master/Spryker/Sniffs/Whitespace/EmptyLinesSniff.php) that @dereuromark provided in the linked issue.
Also added tests for more then one blank line between properties and methods.

Not sure about the licensing

**Comment by markstory** [comment]
@cleptric You need to leave the old author and license tags in-tact.

File: CakePHP/Sniffs/WhiteSpace/EmptyLinesSniff.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ * @link          http://pear.php.net/package/PHP_CodeSniffer_CakePHP
+ * @since         CakePHP CodeSniffer 2.4.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
+/**
+ * Ensures that not more than one blank line occurs
+ *
+ * @author Mark Scherer
+ * @license MIT
```
Comment (ravage84) [inline CakePHP/Sniffs/WhiteSpace/EmptyLinesSniff.php]: URL missing, see https://github.com/cakephp/cakephp-codesniffer/pull/184/files#diff-6bad666c7cf4c732fbc10140a4d6ded0R15

Comment (cleptric) [inline CakePHP/Sniffs/WhiteSpace/EmptyLinesSniff.php]: That was copied from the source https://github.com/spryker/code-sniffer/blob/master/Spryker/Sniffs/Whitespace/EmptyLinesSniff.php

Comment (dereuromark) [inline CakePHP/Sniffs/WhiteSpace/EmptyLinesSniff.php]: If you want you can add the URL of course.

**Comment by cleptric** [comment]
Ping

**Comment by bravo-kernel** [comment]
Great job 🍰

**Pull request merged**